packages/graphql-language-service-server/src/GraphQLLanguageService.ts

```diff
@@ -111,7 +149,7 @@ export class GraphQLLanguageService {
     // Perform syntax diagnostics first, as this doesn't require
     // schema/fragment definitions, even the project configuration.
     let documentHasExtensions = false;
-    const projectConfig = this.getConfigForURI(uri);
+    const projectConfig = this.getProjectForQuery(document, uri);
```

Comment:
can you rename this to `getProjectForDocument` ? just so people don't assume this only works with queries, as it should also work with SDL

Reply:
Updated! I've also resolved the new conflicts but fyi, there's a problem with `cspell` on commit because it finds unknown words in `.vscode/settings.json`.